boards/xtensa/olimex_esp32_evb/olimex_esp32_evb.dts

```diff
@@ -6,7 +6,7 @@
 
 /dts-v1/;
 
-#include <espressif/esp32/esp32_wrover_e_n4r8.dtsi>
+#include <espressif/esp32/esp32_wrover_e_n4r2.dtsi>
```

Comment:
Is it a fixup of previous commit? Which SiP is the right one?

Reply:
This one is tricky since the manufacturer is not specific about the installed Flash or Spiram. So I leave it as is.